### PR TITLE
Speed up pairing by reusing the client

### PR DESCRIPTION
**Pull request by bdraco**

### CODE REVIEW

File: aiohomekit/controller/ble/pairing.py [hunk 1]
```diff
@@ -93,48 +93,43 @@ class BlePairing(AbstractPairing):
     This represents a paired HomeKit IP accessory.
     """
 
-    pairing_id: str
```
Comment (bdraco) [inline aiohomekit/controller/ble/pairing.py:96]: pairing_id looks like cruft

**Review by Jc2k — APPROVED** [review]

**Comment by bdraco** [comment]
```
2022-07-16 17:20:54.609 ERROR (MainThread) [homeassistant.components.homekit_controller.config_flow] Pairing attempt failed with an unhandled exception
Traceback (most recent call last):
  File "/Users/bdraco/home-assistant/homeassistant/components/homekit_controller/config_flow.py", line 424, in async_step_pair
    return await self._entry_from_accessory(pairing)
  File "/Users/bdraco/home-assistant/homeassistant/components/homekit_controller/config_flow.py", line 536, in _entry_from_accessory
    name = await pairing.get_primary_name()
  File "/Users/bdraco/home-assistant/venv/lib/python3.10/site-packages/aiohomekit/controller/abstract.py", line 115, in get_primary_name
    accessories = await self.list_accessories_and_characteristics()
  File "/Users/bdraco/home-assistant/venv/lib/python3.10/site-packages/aiohomekit/controller/ble/pairing.py", line 86, in _async_wrap
    return await func(self, *args, **kwargs)
  File "/Users/bdraco/home-assistant/venv/lib/python3.10/site-packages/aiohomekit/controller/ble/client.py", line 64, in _async_wrap
    return await func(*args, **kwargs)
  File "/Users/bdraco/home-assistant/venv/lib/python3.10/site-packages/aiohomekit/controller/ble/pairing.py", line 405, in list_accessories_and_characteristics
    await self._populate_accessories_and_characteristics()
  File "/Users/bdraco/home-assistant/venv/lib/python3.10/site-packages/aiohomekit/controller/ble/pairing.py", line 475, in _populate_accessories_and_characteristics
    self.description.config_num,
AttributeError: 'NoneType' object has no attribute 'config_num'
```

**Comment by bdraco** [comment]
Shaved about 9s off the pairing on linux 👍

**Comment by bdraco** [comment]
Mac was fast anyways.  Only shaved 3s

**Pull request merged**